### PR TITLE
druid 0.15.1-incubating

### PR DESCRIPTION
**Pull request by chenrui333**
bump the version.

### CODE REVIEW

File: Formula/druid.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Druid < Formula
   desc "High-performance, column-oriented, distributed data store"
   homepage "https://druid.apache.org/"
-  url "http://static.druid.io/artifacts/releases/druid-0.12.3-bin.tar.gz"
-  sha256 "807581d54fa4c5a90eec2a230e2a7fc4c6daf18eb8136009bf36a775d793d6f6"
+  url "https://www.apache.org/dyn/closer.cgi?path=/incubator/druid/0.15.0-incubating/apache-druid-0.15.0-incubating-bin.tar.gz"
```
Comment (chenrui333) [inline Formula/druid.rb]: The download url has been moved.

**Comment by chenrui333** [comment]
19:47:27 Started broker node (93061)
19:47:27 cat: /usr/local/Cellar/druid/0.15.0/libexec/conf-quickstart/druid/broker/jvm.config: No such file or directory
19:47:27 ==> curl -s http://localhost:8082/status
19:47:27 ==> /usr/local/Cellar/druid/0.15.0/bin/druid-broker.sh stop
19:47:27 No broker node to stop
19:47:27 Error: druid: failed
19:47:27 An exception occurred within a child process:
19:47:27   Test::Unit::AssertionFailedError: <0> expected but was
19:47:27 <7>.

**Comment by chenrui333** [comment]
need some time fixing the test phase.

**Comment by SMillerDev** [comment]
Is `incubating` the stable version?

**Comment by chenrui333** [comment]
yeah, `incubating` meaning the project is Apache incubating project, but the release is stable. 

They have rc releases for the project as well.

**Comment by chenrui333** [comment]
Circle back on this.

**Pull request closed without merging**